ocs_ci/utility/ibm_hci.py

```
@@ -62,6 +78,276 @@ def _load_rack_details(self):
+                    f"Verifying BMC connectivity for {node_label} "
+                    f"(trying {len(mgem_candidates)} MGem(s))"
+                )
+
+                node_reachable = False
+                working_mgem = None
+
+                for mgem_ip in mgem_candidates:
+                    for ip_type, bmc_ip in bmc_ips:
+                        log.info(
+                            f"Pinging BMC {ip_type} ({bmc_ip}) for {node_label} "
+                            f"from MGem {mgem_ip}"
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)